src/thunderbots/software/util/refbox_constants.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+enum class RefboxGameState
+{
+    HALT = 0,
+    STOP,
+    NORMAL_START,
+    FORCE_START,
+    PREPARE_KICKOFF_US,
+    PREPARE_KICKOFF_THEM,
+    PREPARE_PENALTY_US,
+    PREPARE_PENALTY_THEM,
+    DIRECT_FREE_US,
+    DIRECT_FREE_THEM,
+    INDIRECT_FREE_US,
+    INDIRECT_FREE_THEM,
+    TIMEOUT_US,
+    TIMEOUT_THEM,
+    GOAL_US,
+    GOAL_THEM,
+    BALL_PLACEMENT_US,
+    BALL_PLACEMENT_THEM
```

Comment:
bump

Reply:
Having corresponding enum-value mapping shouldn't matter to the AI node, because Refbox sends this field as, for example, `BALL_PLACEMENT_BLUE` instead of `BALL_PLACEMENT_US`, and backend handles that translation to US/THEM.

Reply:
I think what @garethellis0 was referring to was that these enums should all be explicitly assigned a value like in `RefboxCommand.msg`, in case the values accidentally get moved out of order (and therefore have different enumeration values). Explicitly defining the enum value should make accidental mismatches less likely.